sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -10,42 +10,45 @@ Testing
 There is a TravisCI project configured to execute on every GitHub push, it can
 be viewed at: https://travis-ci.org/bokeh/bokeh.
 
-To run the just the python unit tests, run the command:
+You can run all available tests (python and JS unit tests, as well as example
+and integration tests) from the top level directory by executing:
 
 .. code-block:: sh
 
-    py.test -m 'not (js or examples)' 
+    py.test
 
+.. note::
+    Currently this script does not support Windows.
 
-To run just the examples, run the command:
+To run just the python unit tests, run the command:
 
 .. code-block:: sh
 
-    py.test -m examples 
+    py.test -m 'not (js or examples or integration)'
```

Comment:
is there some reason we can't tag the unit tests so that `-m unit` (or whatever) is possible? That would be much better.